libc/src/__support/CPP/type_traits/is_function.h

```diff
@@ -17,7 +17,7 @@
 namespace LIBC_NAMESPACE::cpp {
 
 // is_function
-#if LIBC_HAS_BUILTIN(__is_function)
+#if __has_builtin(__is_function)
```

Comment:
We no longer need to include src/__support/macros/config.h here thanks to this change.

Reply:
Done.